pkg/api/apis/operators/install/install.go

```diff
@@ -15,4 +15,4 @@ func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(v1alpha1.AddToScheme(scheme))
 	utilruntime.Must(v1.AddToScheme(scheme))
 	utilruntime.Must(scheme.SetVersionPriority(v1.SchemeGroupVersion, v1alpha1.SchemeGroupVersion))
-}
\ No newline at end of file
+}
```

Comment:
I'm all for newlines at the end of files, but it's good to avoid bundling trivial changes like this into commits that don't otherwise affect a particular file. It makes the commit log a little more useful and helps PR reviewers to focus on the important stuff.

Reply:
Running `gofmt` on the entire project caused this. I did exclude files outside of `test` dir, but this one slipped through. I have configured GoLand to apply `gofmt` only to the files I change. Hopefully, this will not occur in future commits. My apologies!